webapp/src/ts/modules/messages/messages.component.ts

```diff
@@ -40,6 +43,8 @@ export class MessagesComponent implements OnInit, OnDestroy {
     private modalService: ModalService,
     private tourService: TourService,
     private responsiveService: ResponsiveService,
+    private userContactService:UserContactService,
```

Comment:
```suggestion
    private userContactService: UserContactService,
```
Looks like all the other parameters are adding this space, we don't have a rule for this in specific but I think it improves readability.